GrailDiary/GrailDiary/ViewControllers/AddPOIViewController.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  AddPOIViewController.swift
+//  GrailDiary
+//
+//  Created by James Sedlacek on 5/27/20.
+//  Copyright © 2020 James Sedlacek. All rights reserved.
+//
+
+import UIKit
+
+protocol addPOIDelegate {
+    func poiWasAdded(_ poi: POI)
+}
+
+class AddPOIViewController: UIViewController {
+
+    @IBOutlet weak var LocationTextField: UITextField!
+    @IBOutlet weak var CountryTextField: UITextField!
+    @IBOutlet weak var Clue1TextField: UITextField!
+    @IBOutlet weak var Clue2TextField: UITextField!
+    @IBOutlet weak var Clue3TextField: UITextField!
```

Comment:
Remember that IBOutlets are just variables with a special @IBOutlet attribute, so they should start with a lowercase letter as well.